forward_engineering/ddlProvider.js

```diff
@@ -908,7 +939,14 @@ module.exports = (baseProvider, options, app) => {
 		},
 
 		alterTable(data) {
-			const alterTableScript = getAlterEntityScript(templates.alterTableScript, data.nameData);
+			if (data.iceberg) {
+				return '// Dynamic Iceberg tables are currently only supported for CREATE statements. Specifying DYNAMIC ICEBERG in any other command (for example, ALTER DYNAMIC ICEBERG TABLE <name>) results in an error.';
```

Comment:
If the dynamic iceberg table does not support altering, it is worth considering the option to
```
DROP TABLE ...;
CREATE TABLE ...<new props>;
```